src/Management/src/Tracing/TracingServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Steeltoe.Common.Extensions;
+using Steeltoe.Logging;
+
+namespace Steeltoe.Management.Tracing;
+
+public static class TracingServiceCollectionExtensions
+{
+    /// <summary>
+    /// Registers an <see cref="IDynamicMessageProcessor" /> that adds tracing details from <see cref="Activity.Current" /> (if found) to log messages.
```

Comment:
It would be nice to mention that it only works in combination with dynamic logging, i.e., `AddDynamicLogging`/`AddDynamicSerilog`.